orchagent/routeorch.cpp

```diff
@@ -1898,7 +1913,10 @@ bool RouteOrch::addRoutePost(const RouteBulkContext& ctx, const NextHopGroupKey
         }
     }
 
-    m_syncdRoutes[vrf_id][ipPrefix] = nextHops;
```

Comment:
What is this change for? The original change in the master branch does not seem to change this.

Reply:
fixed the cherry-pick error. Thanks for finding